recipes/mmtf-python/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "mmtf-python" %}
+{% set version = "1.0.4" %}
+{% set sha256 = "8aa92bc66b055f952c88f2dc33cd5dd52e39df376bd8ad88cca1cc344b2990da" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - msgpack-python
+
+  run:
+    - python
+    - msgpack-python
+
+test:
+  imports:
+    - mmtf
+    - mmtf.api
+    - mmtf.codecs
+    - mmtf.codecs.decoders
+    - mmtf.codecs.encoders
+    - mmtf.converters
+    - mmtf.tests
+    - mmtf.utils
+
+about:
+  home: http://mmtf.rcsb.org
+  license: Apache-2.0
+  license_family: Apache
+  # will be added in the next version
+  # license_file: LICENSE.txt
```

Comment:
@kain88-de if you do not want to wait for the next release go for @synapticarbors suggestion. Here is a copy-n-pastable version of this suggestion BTW:

First `wget https://raw.githubusercontent.com/rcsb/mmtf-python/master/LICENSE.txt`, then add:

```yaml
 license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
```